src/bindings.rs

```diff
@@ -1,12 +1,17 @@
 // Disable several lints. The auto-generated bindings do not conform to them.
+// TODO: Revisit and revise periodically, especially after changing the MSRV.
```

Comment:
Not ideal and the lints may also depend on the Rust version.

An alternative would be trying to disable all lints for `bindings`.